aligner/u_of_michigan_aligner/u_of_michigan_aligner.wdl

```diff
@@ -12,7 +12,7 @@
 
 workflow TopMedAligner {
 
-  File input_crai_file
+  File? input_crai_file
```

Comment:
If the CRAI files are not needed please remove the crai input and remove the samtools index call

Reply:
@wshands/team, do you how much difference having the CRAI file available or not affects the time required to run the aligner?
If not, it would be worth investigating before removing the CRAI file as an input parameter altogether.
For the projects that I am aware of, the CRAI files are readily available, and if they improve the performance of the aligner, we should continue to be able to provide them as inputs.
If the aligner can be run without the CRAI, but just doesn't perform as well, it is reasonable/good to make the CRAI input file parameter optional.

Reply:
If you compare the performance with and without the CRAI, please be sure NOT to use preemptible instances, so you get comparable results.

When I last ran this workflow a few months ago, it was using preemptible instances by default. It shouldn't be. Google states the preemptible instances are for up to 24 hours, and Alex Baumann said they are not recommended for jobs running more than 4 hours, and actual alignments typically take longer than both these values.

Reply:
In my testing with full-sized CRAM files, including CRAIs didn't seem to have any correlation with runtime, but in fairness that testing is limited.

As for preempts, I am going to keep the default behavior as is in my updated PR because there is some glitch on Terra preventing me from seeing any of my runtime costs, so I can't do meaningful comparisons of cost. Post-align by default does not use preempts and that's the only step that takes over 24 hours on full-size CRAMs, so the preempts that do happen by default (much faster align and pre-align steps) won't cause workflow failure afaik.